Orleans.Streams.Kafka/Config/KafkaStreamOptions.cs

```diff
@@ -120,6 +121,14 @@ public class TopicConfig
 		/// </summary>
 		/// <remarks>1 by default</remarks>
 		public short ReplicationFactor { get; set; } = 1;
+
+		/// <summary>
+		/// If <see cref="RetentionPeriodInMs"/> is set the topic will
+		/// be created with only retain data for this much duration.
+		/// If not set, it'll take default configuration of broker which is 7 days.
+		/// </summary>
+		/// <remarks>7 days by default</remarks>
+		public ulong RetentionPeriodInMs { get; set; } = 604800000;
```

Comment:
can you make this `nullable`? as it is any AutoCreated Topic will have a default retention period of 7 days. I would prefer if there is **_no_** a retention period by default

Reply:
Updated.